packages/corvid-cli/src/apps/pull.js

```diff
@@ -50,6 +50,16 @@ const pullApp = ({ useSsl = true, override = false, move = false } = {}) => ({
         isHeadless
       );
 
+      win.webContents.on("did-finish-load", async () => {
+        const editorLoadingFailed = await win.webContents.executeJavaScript(
+          "window.editorModel === undefined"
```

Comment:
Please check with @EtaiG / Editor folks that this is a legit way to verify if the editor is loaded (maybe @inballa knows?)